x11-packages/virglrenderer/build.sh

```diff
@@ -3,8 +3,9 @@ TERMUX_PKG_DESCRIPTION="A virtual 3D GPU for use inside qemu virtual machines"
 TERMUX_PKG_LICENSE="MIT"
 TERMUX_PKG_MAINTAINER="@termux"
 TERMUX_PKG_VERSION=0.10.4
+TERMUX_PKG_REVISION=1
 TERMUX_PKG_SRCURL=https://gitlab.freedesktop.org/virgl/virglrenderer/-/archive/virglrenderer-${TERMUX_PKG_VERSION}/virglrenderer-virglrenderer-${TERMUX_PKG_VERSION}.tar.gz
 TERMUX_PKG_SHA256=fd9a1b12473f4cda8d87e6ba1a6e5714a24355e16b69ed85df5c21bf48f797fa
-TERMUX_PKG_DEPENDS="libepoxy, libx11"
-TERMUX_PKG_BUILD_DEPENDS="libdrm, xorgproto"
-TERMUX_PKG_EXTRA_CONFIGURE_ARGS="-Dplatforms=glx"
+TERMUX_PKG_DEPENDS="libdrm, libepoxy, libx11"
+TERMUX_PKG_BUILD_DEPENDS="xorgproto"
```

Comment:
`libvirglrenderer.so` has `libgbm.so` as a `DT_NEEDED` entry:

```
$ readelf -d $PREFIX/lib/libvirglrenderer.so | grep NEEDED
  0x0000000000000001 (NEEDED)       Shared library: [libm.so]
  0x0000000000000001 (NEEDED)       Shared library: [libepoxy.so]
  0x0000000000000001 (NEEDED)       Shared library: [libdrm.so]
  0x0000000000000001 (NEEDED)       Shared library: [libgbm.so]
  0x0000000000000001 (NEEDED)       Shared library: [libX11.so]
  0x0000000000000001 (NEEDED)       Shared library: [libc.so]
```

and `$PREFIX/lib/libgbm.so` is provided by `mesa` package. So `virglrenderer` should (run-time) depend on `mesa`. (Note that `mesa` cannot be replaced by `opengl` here.)